Clarification about instance-level methods defined by cattr_accessor (#55328)
rails/rails: guides/source/active_support_core_extensions.md

end
```

> Instance methods are also created for convenience, but they are simply proxies to the underlying class variable. As a result, when an instance sets the value, it modifies the shared class variable, which affects the entire class hierarchy. This is different behaviour when compared with `class_attribute` (see above). Example:

Reviewer: ```suggestion
Instance methods are also created for convenience, but they are simply proxies to the internal value which is shared among the class. As a result, when an instance modifies the value, this affects the entire class hierarchy. This behavior is different than `class_attribute` (see above).

For example:
```

end
```

> Instance methods are also created for convenience, but they are simply proxies to the underlying class variable. As a result, when an instance sets the value, it modifies the shared class variable, which affects the entire class hierarchy. This is different behaviour when compared with `class_attribute` (see above). Here’s an example of the instance-level method in use:

Reviewer 2: I think this is a good improvement, except for the last sentence. The code example is in the middle of a sentence 

> For example given ... we can access `field_error_proc` in views

but the new phrasing doesn't work 

> Here's an example of the instance-level method in use: ... we can access `field_error_proc` in views

Author: Thank you! Come to think of it, perhaps the example should be removed altogether, because:

a) The concept of instance-level methods is mostly self-explanatory. 

b) I find the example distracting because it talks about Views, an unrelated topic, and it's not clear why one would use that technique instead of view helpers, or the myriad of other ways of having instance methods in views. 

c) It seems like `cattr_accessor` is not super encouraged these days because of class variables, so maybe there's no harm in removing the example.

Reviewer: Just my opinion but until we have officially (or somewhat-officially) deprecated something, removing the documentation seems undesireable. But I don't feel comfortable suggesting we rewrite the example here either.

I think the following examples are there to help the reader understand the usage, not that it is a recommended use-case?

It doesn't make sense to show people how something works by monkey-patching Rails, it'd be better to find a more realistic example which follows modern best practices -- but I realize that may be asking a lot of this PR.
